content/realtime/channels/channel-parameters/overview.textile

```diff
@@ -0,0 +1,82 @@
+---
+title: Channel Parameters
+section: realtime
+index: 1
+jump_to:
+  Help with:
+    - Overview#overview
+    - Supported channel parameters#supported-parameters
+    - Using channel parameters#using-parameters
+    - Using channel parameters with non-Ably transports#using-parameters-with-other-transports
+    - Next steps#next-steps
+---
+
+h2(#overview). Overview
+
+Channel parameters allow a client to set the properties of a channel. A set of channel parameters is a set of key/value pairs, where both keys and values are strings; the keys correspond to specific features that Ably defines.
+
+The methods provided for specifying channel parameters are outlined below.
+
+h2(#supported-parameters). Supported channel parameters
+
+- rewind := Used to request that an attachment start from a given number of messages or point in time in the past. See "rewind":/realtime/channels/channel-parameters/rewind for more information.
+- delta := Used to request that data payloads should be sent as deltas to the previous payload. See "deltas":/realtime/channels/channel-parameters/deltas for more information.
+
+h2(#using-parameters). Using channel parameters
```

Comment:
The section below describes using channel parameters with non-Ably transports, yet this one does not state this only works with Ably transports.

Reply:
@paddybyers can you help confirm what the correct terminology for this title would be? I'm guessing 'Using channel parameters with Ably transports' isn't quite correct?

Reply:
Addressed with update